tendrl/commons/flows/create_cluster/utils.py

```diff
@@ -178,3 +180,83 @@ def gluster_create_ssh_setup_jobs(parameters, skip_current_node=False):
         )
     return ssh_job_ids
 
+
+def accuire_node_lock(parameters, flow_name):
+    Event(
+        Message(
+            job_id=parameters['job_id'],
+            flow_id=parameters['flow_id'],
+            priority="info",
+            publisher=NS.publisher_id,
+            payload={
+                "message": "Checking if nodes are not locked already"
+            }
+        )
+    )
+    fail = False
+    for node in parameters['Node[]']:
```

Comment:
This for loop logic is incorrect, First check if "any" node is locked, if yes then raise exception, why are you locking while checking this?